### PR TITLE
Codeclimate/quartal2 week2

### PR DESCRIPTION
**Pull request by firmanJS**
## OVERVIEW

reduce code smells and duplication code

### CODE REVIEW

**Review by khihadysucahyo — APPROVED** [review]

File: test/user.test.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-// eslint-disable-next-line no-undef
-describe('api/v1/user ', () => {
-  // eslint-disable-next-line no-undef
-  it('not found page', (done) => {
-    request(dummyUrl)
-      .get('/users')
-      .expect(404, done);
-  });
-});
+const request = require('supertest');
```
Comment (sandisunandar99) [inline test/user.test.js:1]: mang jang test case mending di pisah branch wae,, jadi siga kieu test/test-users-login

Comment (firmanJS) [reply test/user.test.js:1]: kabawa sorry :smile:

Comment (firmanJS) [reply test/user.test.js:1]: gas jangan lupa approve @sandisunandar99

Comment (sandisunandar99) [reply test/user.test.js:1]: oraitt... gass,, :+1:

**Review by sandisunandar99 — APPROVED** [review]

**Pull request merged**